LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Any thoughts on MIT vs. GNU to consider?

Reply:
Pros and Cons? MIT is the one I'm most familiar with

Reply:
GNU makes sure if other people distribute it it's still open source

Reply:
But also should it be open source???

Reply:
Has to be if we want all the freebies for github (e.g. organisations). DApps in general are opensource and its security by obscurity if we tried to hide the code.